nototools/unicode_data.py

```diff
@@ -327,6 +327,9 @@ def create_script_to_chars():
 def defined_characters(version=None, scr=None):
     """Returns the set of all defined characters in the Unicode Standard."""
     load_data()
+    # handle common error where version is passed as string, the age test
+    # will always pass
+    version = float(version)
```

Comment:
Pfffft!

There's no code change so small that it doesn't require code review... at least in my case.